interfaces/Libraries.ts

```diff
@@ -0,0 +1,184 @@
+
+/**
+ * Add your new library here.
+ */
+export enum ModelLibrary {
```

Comment:
Could the `ModelLibrary` be built from the `MODEL_LIBRARIES_UI_ELEMENTS` so that users contributors have less to fill? It seems to be close to 1-to-1 with the `btnLabel` in the `MODEL_LIBRARIES_UI_ELEMENTS`

Reply:
I guess we could remove the btnLabel if indeed we expect it to already be the same string.